sssd_test_framework/utils/sssd.py

```diff
@@ -783,3 +783,54 @@ def autofs(self) -> None:
         """
         self.sssd.authselect.select("sssd")
         self.sssd.enable_responder("autofs")
+
+    def proxy(self,
+              host: MultihostHost,
+              domain: str = None,
+              proxy: str = None,
```

Comment:
we can use type `Literal["files", "ldap"]` since we expect specific value. None does not make sense since you don't default to anything so lets make it a required parameter.

Reply:
This was not resolved.